action.yml

```diff
@@ -0,0 +1,23 @@
+name: Jira Pull Request
+author: On Running
+description: Add Jira ticket to PR title / add links to PR description
+inputs:
+  github-token:
+    description: The GitHub token used to create an authenticated client
+    default: ${{ github.token }}
+    required: false
+  jira-account:
+    description: Subdomain used for jira link (i.e. foobar => https://foobar.atlassian.net/browse/ABC-123)
+    required: true
+  ticket-regex:
+    description: Regex to match jira ticket in branch name (i.e. ^ABC-\d+)
+    required: true
+  clean-title-regex:
+    description: Regex used to delete text from PR title
+    required: false
+  preview-link:
+    description: Preview link to add to PR description (i.e. https://preview.example.com)
+    required: false
+runs:
+  using: node16
```

Comment:
Is this the node base image?
If so, we're running CI with node 14.x, shouldn't we use the same node version for the runtime ?

Reply:
@lneves75 According to [the documentation](https://docs.github.com/en/actions/creating-actions/metadata-syntax-for-github-actions#runsusing), we can only specify either `node12` or `node16`. That being said, nothing prevent us from using Node 16 in this repository which is anyway the LTS version.

Reply:
So now the project uses Node 16.13.2 👍